uyuni/common-libs/uyuni-common-libs.spec

```diff
@@ -67,6 +67,7 @@ BuildRequires:  python-devel
 %else
 BuildRequires:  python2-devel
 %endif
+Recommends:     zchunk
```

Comment:
I'm curious / may lack context: why `Recommends` and not `Requires`?

Reply:
It's not available on SLE yet, but it's present in Leap

Reply:
https://github.com/uyuni-project/uyuni/pull/4073#issuecomment-896741633

Reply:
Okay thanks for the clarification